src/main/scala/nl/knaw/dans/easy/fedoratobag/FileInfo.scala

```diff
@@ -61,10 +61,10 @@ case class FileInfo(fedoraFileId: String,
 }
 
 object FileInfo extends DebugEnhancedLogging {
-  private val nonAllowedCharacters = List(':', '*', '?', '"', '<', '>', '|', ';', '#')
+  private val allowedCharacters =(('a' to 'z') ++ ('A' to 'Z') ++ ('0' to '9') ++ List('_', '-', '.', '\\', '/', ' ')).toSet
```

Comment:
The backslash surprises me. On the other hand: doesn't that imply the following character is supposed to be a space? That constraint is not checked.

Reply:
That is a good point, Joke, about the space after backslash.  On the other hand, the backslash and the space after it is created by the (operating?) system, not by our code. So, we could just trust that it is ok.  And if there would after all be something else after the backslash, that doesn't hurt either, does it?

Reply:
it's more complicated than i thought ... these chars are allowed in the path. in the file name itself, more chars are allowed. 
i'll push a new commit to reflect this